sql/README

```diff
@@ -1,23 +1,23 @@
-*** Tools for users
+\*\*\* Tools for users
```

Comment:
The changes in this file should be undone - it's plaintext, not markdown, and the indents were used on purpose.

Reply:
I wonder if we should make it MD for consistency...